pkg/types/series_float64.go

```diff
@@ -52,13 +58,17 @@ func (f *Float64Series) Bind(source Float64Source, target Float64Calculator) {
 	if canTruncate {
 		c = func(x float64) {
 			y := target.Calculate(x)
-			target.PushAndEmit(y)
-			trc.Truncate()
+			if !math.IsNaN(y) {
```

Comment:
I don't think Float64Series should do such filtering.
A Filter function or Map function will do the work.

Reply:
Ok~

Reply:
Updated